### ISSUE TITLE
resume job ?

### ISSUE THREAD

**Issue by ptranvan** [lbcb-sci/ra#6]
Hi,

My job failed due to cluster limit. 
Is it possible to resume a job ?

I was at this stage :

```
[M::main] Version: 2.14-r892-dirty
[M::main] CMD: ra/vendor/minimap2/minimap2 -t 53 -x ava-ont Tpa_nanopore.fastq Tpa_nanopore.fastq
[M::main] Real time: 269685.254 sec; CPU: 11681533.570 sec; Peak RSS: 57.985 GB
[Ra::run] preconstruction stage
[rala::Graph::initialize] loaded sequences
```

**Comment by rvaser** [comment]
Hello,
the answer is maybe.  If the working directory is still there, you could modify the script so that it skips overlap step and uses the old directory. Comment out lines https://github.com/lbcb-sci/ra/blob/master/scripts/ra.py#L53-L68, add the full path to the old working directory here https://github.com/lbcb-sci/ra/blob/master/scripts/ra.py#L25, and run `cmake` again.

Best regards,
Robert

**Comment by ptranvan** [comment]
Thanks for your answer. Yes I still have the old directory. 

Before it stops, the step `[rala::Graph::initialize] loaded sequences` stucked for almost 3 days and the file `preconstruction.fasta` was empty. 

Is it normal ? I have 80X nanopore data and my genome size is 1.2G

**Comment by rvaser** [comment]
Yeah, I had a problem in parsing gzipped files and did not have time to update the old repository. However, this is resolved in our upgraded assembler: https://github.com/lbcb-sci/raven. For both Ra and Raven, you will need RAM equal to the size of sequences plus some epsilon.
